src/persistence/rolle.entity.ts

```diff
@@ -1,9 +1,11 @@
 import { TimestampedEntity } from './timestamped.entity.js';
-import { Entity, ManyToOne } from '@mikro-orm/core';
-import { SchulstrukturknotenEntity } from './schulstrukturknoten.entity.js';
+import { Entity, Property } from '@mikro-orm/core';
 
 @Entity({ tableName: 'rolle' })
 export class RolleEntity extends TimestampedEntity<RolleEntity, 'id'> {
-    @ManyToOne()
-    public administeredBy!: SchulstrukturknotenEntity;
+    /**
+     * Points to Schulstrukturknoten
+     */
+    @Property()
+    public administeredBy!: string;
```

Comment:
I'm not sure if it is a good idea to not name it "schulstrukturknoten". I know you want to show the relation here, but the inconsistency in naming the variable different than in other Entities bothers me more... Is there another way to show the special relation?